nautobot/ipam/migrations/0020_aggregate_to_prefix_data_migration.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime, time
+
+from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import migrations
+from django.utils.timezone import make_aware
+
+from nautobot.ipam import choices
+
+
+def migrate_aggregate_to_prefix(apps, schema_editor):
+
+    Prefix = apps.get_model("ipam", "Prefix")
+    Aggregate = apps.get_model("ipam", "Aggregate")
+    Status = apps.get_model("extras", "Status")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    prefix_ct = ContentType.objects.get_for_model(Prefix)
+
+    try:
+        prefix_default_status = Status.objects.get(content_types=prefix_ct, slug="active")
+    except ObjectDoesNotExist:
+        prefix_default_status = Status.objects.filter(content_types=prefix_ct).first()
+
+    for instance in Aggregate.objects.all():
+        # convert date to datetime
+        date_allocated = instance.date_added
+        if date_allocated is not None:
+            date_allocated = datetime.combine(date_allocated, time.min)
+            if settings.USE_TZ:
+                date_allocated = make_aware(date_allocated)
```

Comment:
I'll turn your question back around at you  :-) Does this make for 00:00 local time, or 00:00 UTC?

Reply:
If I don't set a time zone it gets set to midnight UTC:

```
nautobot=# select date_allocated from ipam_prefix;
     date_allocated     
------------------------
 2023-02-21 00:00:00+00
(1 row)
```

If I set `NAUTOBOT_TIME_ZONE="America/Chicago"` it gets set to midnight in Chicago (but still stores it in the DB as UTC)
```
nautobot=# select date_allocated from ipam_prefix;
     date_allocated     
------------------------
 2023-02-21 06:00:00+00
(1 row)
```

That seems acceptable to me?

Reply:
It's a bit inconsistent with #3316, but as long as we document it clearly I think it's probably fine.